.github/workflows/poll-starter-workflows.yml

```diff
@@ -0,0 +1,80 @@
+name: Poll Starter Workflows
+
+on:
+  schedule:
+    - cron: "0 */6 * * *"
+  workflow_dispatch:
+
+env:
+  STARTER_WORKFLOW_PATH: "./resources/yaml/"
+  STARTER_WORKFLOW_REPO_BRANCH: "partner_templates"
+
+jobs:
+  poll:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download run information
+        uses: dawidd6/action-download-artifact@v2
+        continue-on-error: true
+        id: download
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          workflow: poll-starter.yml
```

Comment:
Think this should be `poll-starter-workflows.yml`?

Reply:
good catch! will fix